test/arithmetic/floating_point/floating_point_conversion_test.dart

```diff
@@ -0,0 +1,201 @@
+import 'dart:convert';
+import 'dart:math';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:test/test.dart';
+import 'package:rohd/rohd.dart';
+
+void main() {
+  const fp32ExponentWidth = 8;
```

Comment:
this PR should provide these constants for reference:
https://github.com/intel/rohd-hcl/pull/110

although, why test these specifically instead of a handful of arbitrary sizes?